docs/source/developer/contributing.rst

```diff
@@ -207,6 +207,12 @@ in the test runner, there is a ``--log`` flag for convenience::
 
     $ python -m numba.runtests --log
 
+To enable :ref:`runtime type-checking <type_anno_check>`, set environment
```

Comment:
Should we also tell people how to get/install typeguard and add it to the list of optional packages in the dependency list (this seems like a similar use case to e.g. `graphviz` for CFG debugging)?

Reply:
yes, i'll add that